blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,119 @@
+
+### fwupd
+
+fwupd is an open-source daemon that manages the firmware updates of various
+devices. You will surely benefit from using fwupd if you are interested in
+automating the firmware update process on your devices and reducing the risk of
+security vulnerabilities. fwupd is compatible with a wide range of devices,
+including laptops, desktops, and IoT devices. Our team can help you seamlessly
+integrate into the fwupd ecosystem across a variety of platforms and operating
+systems. Say goodbye to clunky update processes and hello to a streamlined,
+hassle-free experience with fwupd!
+
+<!-- List of contribution-->
+
+### Yocto
+
+If you're looking for a tailored Linux-based operating system that perfectly
+meets your unique requirements and security needs, Yocto is the open-source
+project for you. As a comprehensive suite of tools and templates, Yocto
+provides the flexibility and customization you need to create a bespoke
+solution for your device.
+
+By partnering with our team, we can help you leverage the power of Yocto and
+build a custom Linux distribution that fully aligns with your vision. From
+feature-rich IoT devices to mission-critical servers, we've got you covered.
+Let us create a personalized solution that meets your exact specifications
+and takes your device's capabilities to the next level.
+
+<!-- List of contribution-->
+
+### Trenchboot
+
+Trenchboot is a game-changing open-source project that provides a powerful set
+of tools for measuring and attesting to the trustworthiness of a system during
+the boot process. If you're looking for secure boot processes for your devices
+- be it IoT devices, servers, or laptops - Trenchboot is the solution you've
+been searching for. Our team is equipped to enable Trenchboot for you so that
+you can measure and attest to the integrity of your system's boot process,
+and enjoy enhanced protection against attacks and device security breaches.
+Let us help you safeguard your devices with Trenchboot.
+
+<!-- List of contribution-->
+
+## Summary
+
+If you're looking for expert guidance on open-source projects such as Coreboot,
+fwupd, Yocto, and Trenchboot, our team is here to help. We are committed to
+helping our clients achieve their security and customization goals through
+our open-source expertise, and we invite you to join us on this journey.
+[We'd love to discuss the details](https://3mdeb.com/contact/)
+of how we can work together to bring your project to the next level.
+
+If you are passionate about these topics, we also welcome you to join our
+recruitment process and become a part of our team.
+[Check here](https://3mdeb.com/careers/) possible career paths.
+
+Additionally, we invite everyone to join our community and
+[matrix space](https://matrix.to/#/%23dasharo-osf-vpub:matrix.org),
+where you can connect with like-minded individuals and stay up-to-date with the
+latest news in the open-source world.
+Don't forget to mark your calendars for
```

Comment:
I like what you wrote in https://blog.3mdeb.com/2023/2023-01-30-3mdeb-insights-winter-2023/

In paragraphs:
- Dasharo User Group (DUG) `#1`
- Dasharo Developers vPub 0x6

Why not have the same two paragraphs but for `#2` and 0x6 with a small summary of the previous DUG and some inspiration for what can happen next time?

Reply:
One paragraph of summary of DUG#1 and vPub 0x6 would be appreciated.

Reply:
I added summary 8e27f10b6db61e09c74d621776c66c05c8d031ad